scripts/rollup/results.json

```diff
@@ -4,29 +4,29 @@
       "filename": "react.development.js",
       "bundleType": "UMD_DEV",
       "packageName": "react",
-      "size": 95936,
-      "gzip": 25258
+      "size": 98406,
+      "gzip": 25783
```

Comment:
Revert this file?

Reply:
We should just delete it and stop generating it on local builds :P